spyder/app/mainwindow.py

```diff
@@ -2715,6 +2715,7 @@ def report_issue(self, body=None, title=None, open_webpage=False):
         if body is None:
             from spyder.widgets.reporterror import SpyderErrorDialog
             report_dlg = SpyderErrorDialog(self, is_report=True)
+            report_dlg.set_color_scheme(CONF.get('appearance', 'selected'))

```

Comment:
This is not the only place where we create an instance of SpyderErrorDialog. There's also one in `spyder/plugins/console/plugin.py`.

Reply:
I will fix it there! thanks!

Reply:
Fixed @ccordoba12